src/frontend/src/lib/derived/token.derived.ts

```diff
@@ -29,7 +34,20 @@ export const tokenDecimals: Readable<number | undefined> = derived(
 	([$token]) => $token?.decimals
 );
 
-export const tokenToggleable: Readable<boolean> = derived(
-	[token],
-	([$token]) => nonNullish($token) && 'enabled' in $token
-);
+export const tokenToggleable: Readable<boolean> = derived([token], ([$token]) => {
```

Comment:
since we are here, some tests for this derived?

Reply:
I will write some tests for this function in a separate PR